apis/management.cattle.io/v3/cluster_types.go

```diff
@@ -246,3 +243,9 @@ type RotateCertificateInput struct {
 	CACertificates bool     `json:"caCertificates,omitempty"`
 	Services       []string `json:"services,omitempty" norman:"type=enum,options=etcd|kubelet|kube-apiserver|kube-proxy|kube-scheduler|kube-controller-manager"`
 }
+
+type LocalClusterAuthEndpoint struct {
+	Enabled bool   `json:"enabled" norman:"default=true"`
```

Comment:
i really, really think this is supposed to be `*bool`

Reply:
As per conversation, `default=true` has been removed so Enabled will default to false.